ui/src/pages/ResultsPage.tsx

```diff
@@ -4684,7 +4683,7 @@ const CodeTabContent = (props: {
 		}
 
 		try {
-			exportToJson(type, data);
+			formatters.exportToJson(type, data);
```

Comment:
called within `formatters` instead of directly so we can mock & spy on it in unit tests